### DISCUSSION TITLE
am_gang_call script and its uses

### DISCUSSION THREAD

**Discussion by Tayipc38** [YimMenu/YimMenu#1215] [Ideas]
There are several ways of interacting with players by sending them certain attackers. Which by my observations happen via a game script called am_gang_call. Some of these interactions are Hit squads, Strike squads, Merryweather etc. Currently I am trying to make the game initiate a Strike squad of any type on any player. By hooking some natives such as START_NEW_SCRIPT_WITH_ARGS I have found that the script is launched via the freemode script from the facility (am_mp_defunct_base script). 

Suspected area where this happens:
![image](https://user-images.githubusercontent.com/79384354/230790093-f45ab151-7491-4963-a702-f59f4cc2e171.png)

The first argument is set to 24 suggesting the script 'am_gang_call' which is fetched from another function with a rather large switch case containing a lot of script names.

a lil snippet:
![image](https://user-images.githubusercontent.com/79384354/230790215-6b684652-56d1-4dbf-a015-c0e93df35319.png)

So far I've only been able to decipher the 18nth argument of the script which pertains the player index of the target.
The complexity of the game scripts and how all of this is achieved has proven a tad bit too difficult for me, hence why I just decided to open a discussion in hopes of someone taking an interest.

**Comment by maybegreat48** [comment]
Setting the service and the target to your local player's [GPBD_FM](https://github.com/Yimura/GTAV-Classes/blob/master/script/globals/GPBD_FM.hpp#L581) should cause the freemode host to start the script

**Reply by Tayipc38** [reply]
That does indeed start the script for the mercenaries, would a version of that also be responsible for the strike team? Like a global or something?

**Reply by maybegreat48** [reply]
It appears that setting `StrikeTeam.Target` along with the mercenaries service should send a strike team instead. The level ranges from one to three, with three spawning the most well-armed enemies
